.circleci/publish-step-2-verdaccio.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+source ./.circleci/codebuild-checkout.sh
+
```

Comment:
This is unexpected.
In CCI `.circleci/publish-step-1-set-versions.sh` adds a commit to local repo that is NOT pushed to git.
That commit is retained in other jobs because we cache result of `build` step, including `.git` directory which preserves that information. We should not need to checkout everywhere if we we cache workspace properly.

This is concern for other files that attempt to checkout after `build` step.